cabal-install/bootstrap.sh

```diff
@@ -213,8 +213,16 @@ OLD_LOCALE_VER="1.0.0.7"; OLD_LOCALE_VER_REGEXP="1\.0\.?"
                        # >=1.0.0.0 && <1.1
 BYTEABLE_VER="0.1.1";  BYTEABLE_VER_REGEXP="0\.?"
                        # 0.1.1
-CRYPTOHASH_VER="0.11.7"; CRYPTOHASH_VER_REGEXP="0\.11\.?"
+MEMORY_VER="0.12";     MEMORY_VER_REGEXP="0\."
+                       # 0.*
+CRYPTONITE_VER="0.15"; CRYPTONITE_VER_REGEXP="0\."
+                       # 0.*
+CRYPTOHASH_VER="0.11.9"; CRYPTOHASH_VER_REGEXP="0\.11\.?"
```

Comment:
why was `cryptohash` updated? this way you've pulled two unneeded additional dependencies for no benefit